backend/SS_Position/core.py

```diff
@@ -13,6 +13,9 @@ def processMessage(player, message):
 def processMovement(player, key):
     baseX,baseY = redisWorker.getPlayerPostition(player)
     x,y = baseX,baseY
+    door_closed = ""
+    if mapWorker.processOpenable(x,y) == 'c':
+        door_closed = 'c' + " " + str(x) + " " + str(y)
```

Comment:
Why this code isn't in separate function? It would be great to move it to function, for example, closeDoor, where I would send x and y of the door.